app/models/job/Job.scala

```diff
@@ -66,6 +67,10 @@ case class Job(
           }.getOrElse(datasetName.map(name => s"datasets/$organizationId/$name/view"))
         case JobCommand.export_tiff | JobCommand.render_animation =>
           Some(s"/api/jobs/${this._id}/export")
+        case JobCommand.infer_neurons if this.argAsBooleanOpt("do_evaluation").getOrElse(false) =>
+          returnValue.map { resultAnnotationLink =>
+            resultAnnotationLink
+          }
```

Comment:
While trying to open the created evaluation annotation I just noticed a little bug:

As a result I got the following link in the jobs page: 
`http://localhost:9000/datasets/sample_organization//annotations/Explorational/678a1d535b0100aa21602e08/view`
This link includes a double `/` which needs to be removed. Likely for consistency sake it should be removed int the worker job as else this logic needs to be implemented in the backend here or in the frontend. Fixing this in the frontend or backend might lead to an "unexpected" special case for infer neuron jobs that have an evaluation. Fixing this in the worker has the advantage that the "special case is directly handled where it is created" IMO. Feel free to disagree

Reply:
Oh wait, I misread the link. The `/datasets/sample_organization/` part needs to be removed I'll give this a check where to best implement this. 
My quick guess would be fixing this in the frontend

Reply:
> My quick guess would be fixing this in the frontend

Mööööppp! Wrong :red_circle: 

The fix is to replace `this.argAsBooleanOpt("do_evaluation")` with `this.argAsBooleanOpt("do_split_merger_evaluation")`. This line simply was forgotten during the raming of the  `do_evaluation` variable. This should fix all the things :tm: